lib/authsense.ex

```diff
@@ -73,6 +73,19 @@ defmodule Authsense do
   things: see [Recipes](recipes.html).
   """
 
+  if Application.get_all_env(:authsense) == [] do
+    raise """
+    Please configure Authsense.
+
+    Example configuration:
+      config :authsense, MyApp.User,
+        repo: MyApp.Repo
+        identity_field: :email,
+        password_field: :password,
+        hashed_password_field: :hashed_password,
```

Comment:
the last 3 fields are optional, maybe it can be left out for brevity and just link to the docs? That is:

> See http://hexdocs.pm/authsense for more configuration options.